test/unit/sync_checker/translations_check_test.rb

```diff
@@ -0,0 +1,129 @@
+require 'minitest/autorun'
+require 'mocha/setup'
+require 'active_support'
+require 'active_support/json'
+require 'active_support/core_ext'
+require_relative '../../../lib/sync_checker/checks/translations_check'
+
+class TranslationsCheckTest < Minitest::Test
```

Comment:
Ditto the above comment it'd be good to add a test to document what happens when `expected_locales` is nil